src/OpenLoco/src/Input/MouseInput.cpp

```diff
@@ -1666,38 +1657,38 @@ namespace OpenLoco::Input
 
     Ui::Point getMouseLocation()
     {
-        return Ui::Point(_cursorX, _cursorY);
+        return Ui::Point(static_cast<int16_t>(_cursor->x), static_cast<int16_t>(_cursor->y));
     }
 
     Ui::Point getMouseLocation2()
     {
-        return Ui::Point(_cursorX2, _cursorY2);
+        return Ui::Point(static_cast<int16_t>(_cursor2->x), static_cast<int16_t>(_cursor2->y));
     }
 
     Ui::Point getTooltipMouseLocation()
     {
-        return Ui::Point(_tooltipCursorX, _tooltipCursorY);
+        return Ui::Point(_tooltipCursor->x, _tooltipCursor->y);
```

Comment:
_tooltipCursor is already a Ui::Point now, so perhaps we should just return that?

Reply:
Oh yea..